workflow/rocoto/tasks.py

```diff
@@ -136,7 +136,7 @@ def _get_forecast_hours(run, config, component='atmos') -> List[str]:
         # Make a local copy of the config to avoid modifying the original
         local_config = config.copy()
         # Ocean/Ice components do not have a HF output option like the atmosphere
-        if component in ['ocean', 'ice']:
+        if component in ['ocean', 'ice', 'wave']:
```

Comment:
Waves do have a HF setting, and it is set down below, so change it for waves here is not needed.
```suggestion
        if component in ['ocean', 'ice']:
```